packages/x-date-pickers/src/locales/faIR.ts

```diff
@@ -45,18 +50,18 @@ const faIRPickers: Partial<PickersLocaleText<any>> = {
   // Clock labels
   clockLabelText: (view, time, adapter) =>
     ` را انتخاب کنید ${timeViews[view]}. ${time === null ? 'هیچ ساعتی انتخاب نشده است' : `ساعت انتخاب ${adapter.format(time, 'fullTime')} می باشد`}`,
-  hoursClockNumberText: (hours) => `${hours} ساعت‌ها`,
-  minutesClockNumberText: (minutes) => `${minutes} دقیقه‌ها`,
-  secondsClockNumberText: (seconds) => `${seconds} ثانیه‌ها`,
+  hoursClockNumberText: (hours) => `${latinToPersianDigits(hours)} ساعت‌ها`,
+  minutesClockNumberText: (minutes) => `${latinToPersianDigits(minutes)} دقیقه‌ها`,
+  secondsClockNumberText: (seconds) => `${latinToPersianDigits(seconds)} ثانیه‌ها`,
 
   // Digital clock labels
   selectViewText: (view) => ` را انتخاب کنید ${timeViews[view]}`,
 
   // Calendar labels
   calendarWeekNumberHeaderLabel: 'عدد هفته',
-  calendarWeekNumberHeaderText: '#',
-  calendarWeekNumberAriaLabelText: (weekNumber) => `هفته ${weekNumber}`,
-  calendarWeekNumberText: (weekNumber) => `${weekNumber}`,
+  calendarWeekNumberHeaderText: latinToPersianDigits('#'),
```

Comment:
This seems to be a mistahe. The `latinToPersianDigits` will do nothing to `'#'`

Reply:
Yeah, thanks for the heads up